Objective-C/Tests/VectorSearchTest.m

```diff
@@ -0,0 +1,142 @@
+
+@interface VectorSearchTest : CBLTestCase
+
+@end
+
+@implementation VectorSearchTest
+
+- (void) setUp {
+    [super setUp];
+}
+
+- (void) tearDown {
+    [super tearDown];
+}
+
+- (void) initDB {
+    NSError* error;
+    CBLDatabaseConfiguration* config = [[CBLDatabaseConfiguration alloc] init];
+    config.directory = self.directory;
+    
+    NSString* path = [self databasePath: @"words_db.cblite2" inDirectory: @"vectorsearch"];
+    [CBLDatabase copyFromPath:path toDatabase: kDatabaseName withConfig: config error: &error];
+
+    [self openDB];
+}
+
+- (NSString*) databasePath: (NSString*)fileName inDirectory: (NSString*)dir {
```

Comment:
Better to move this method to CBLTestCase so it can be shared with MigrationTest.